hubmap_api_py_client/external.py

```diff
@@ -202,6 +202,14 @@ def _add_method(output_type, ResultsSetSubclass, args_type, doc):
             genomic_modality=genomic_modality,
             logical_operator=logical_operator,
             ResultsSetSubclass=ResultsSetSubclass),
+        'min_cells': lambda self, where=None, has=None,
+            genomic_modality=None, logical_operator=None, min_cell_percentage=None:
+        self._query(
+            input_type=where, output_type=output_type, has=has,
+            genomic_modality=genomic_modality,
+            logical_operator=logical_operator,
+            min_cell_percentage=min_cell_percentage,
+            ResultsSetSubclass=ResultsSetSubclass),
         'where_has': lambda self, where=None, has=None:
```

Comment:
Below, you replace `where_has` with `min_cells`... Should `where_has` be deleted here, or does it still have a role?

Reply:
Where_has still has a place with protein queries